src/lib/AnnotationControlsFSM.ts

```diff
@@ -33,7 +39,7 @@ export const stateModeMap = {
 export default class AnnotationControlsFSM {
     private currentState = AnnotationState.NONE;
 
-    public transition = (input: AnnotationInput, mode: AnnotationMode): AnnotationMode => {
+    public transition = (input: AnnotationInput, mode: AnnotationMode = AnnotationMode.NONE): AnnotationMode => {
```

Comment:
Having mode default to `none` seems ripe for bugs to be introduced. Would it work if it was just optionally undefined?

Reply:
The state in FSM cannot be `undefined`, so if mode is `undefined` and state transition depends on mode, the state has to be `none` which is the same as default mode to `none`.